controllers/employeeDashboard.controller.js

```diff
@@ -0,0 +1,61 @@
+const User = require("./../modules/user.module");
```

Comment:
the title should be user.controller.js, as it handles users, not just employees or managers